### DISCUSSION TITLE
cannot run run realistic

### DISCUSSION THREAD

**Discussion by cakueayam** [lllyasviel/Fooocus#3168] [Q&A] [answered]
hi, i cannot run run_realistic bat, and it shows like this
![Screenshot 2024-06-20 103016](https://github.com/lllyasviel/Fooocus/assets/52396142/2b6be47e-07f5-4ca1-ab33-4c1841eb1952)

**Comment by cakueayam** [comment]
and why the speed is suddenly reduced to around 280kb/s?

**Comment by mashb1t** [comment, marked as answer]
This has nothing to do with Fooocus. It's recommended to use 7zip to extract the zip file, so complications are minimized on your machine.
